docs/getting_started/basics/execute.md

```diff
@@ -34,22 +36,65 @@ The "Execute Next" action will compute the next dirty node present in the workfl
 </figure>
 
 
-## ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
+### ![Clear Execution Button](images/clear_all_default.svg){style="width:5%"} Clear Execution
 
-The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed. 
+The "Clear Execution" action will set all the nodes in the workflow to dirty. All **connected** inputs and output values will be cleared to their default values. This forces Shift to recompute all nodes the next time the workflow has to be executed.
 
 <figure>
       <img src="images/clear_execution.gif" alt="Clear execution">
       <figcaption><b>Figure 4</b>: Clear the executed status from all nodes.</figcaption>
 </figure>
 
 
-## ![Live Execution Button](images/execute_live_default.svg){style="width:5%"} Live Execution
+### ![Live Execution Button](images/execute_live_default.svg){style="width:5%"} Live Execution
 
 The "Live Execution" action allows to trigger the execution of the whole workflow whenever a change to any node is done. While the "Live Execution" is turned on, the other execution buttons are disabled. To re-enable all the execution buttons, turn Live Execution off.
 
 <figure>
-      <img src="images/live_execution.gif" alt="Live Execution">
+      <img src="images/stopping_execution.gif" alt="Live Execution">
```

Comment:
I guess this should be live_execution.gif?

Reply:
<3
Changed it, thank you!